_aggregations/bucket/composite.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Composite
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 20
+has_children: true
+---
+
+# Composite
+
+The `composite` aggregation is a multi-bucket aggregation that creates composite buckets from different sources. It is useful for efficiently paginating multi-level aggregations and retrieving all buckets. Composite buckets are built from combinations of values extracted from documents for each specified source field.
+
+## Syntax
```

Comment:
**Technical reviewer:** Please confirm this example is relevant to an OpenSearch user. I tested the example using Dev Tools. If another example is more appropriate, please replace the draft example with your example. Thank you.

Reply:
@msfroh @bowenlan-amzn @sandeshkr419 Would one of you be available to technical review this PR? The doc team is closing content gaps for aggregations and need a technical review before we can published the doc. Please note that I used Amazon Q to assist in code example generation and Dev Tools to test those examples. Thank you